packages/ui/src/components/Snippet/index.tsx

```diff
@@ -212,13 +239,26 @@ export const Snippet = ({
     >
       <StyledStack>
         {hasShowMoreButton ? (
-          <Expandable minHeight={120} opened={showMore}>
-            <CodeContent prefix={prefix} multiline={multiline} lines={lines}>
+          // Height = number of rows * height of a line + padding
+          <Expandable minHeight={rows * LINE_HEIGHT + 32} opened={showMore}>
+            <CodeContent
+              prefix={prefix}
+              multiline={multiline}
+              lines={lines}
+              noExpandable={noExpandable}
+              rows={rows}
+            >
               {children}
             </CodeContent>
           </Expandable>
         ) : (
-          <CodeContent prefix={prefix} multiline={multiline} lines={lines}>
+          <CodeContent
+            prefix={prefix}
+            multiline={multiline}
+            lines={lines}
+            noExpandable={noExpandable}
+            rows={rows}
+          >
```

Comment:
There is a small issue with the copy button container, it seems that we applied a background (I don't know why) so when the scrollbar appear it goes under. Can you remove the ButtonContainer background so it doesn't do that?

https://github.com/user-attachments/assets/4ecdd660-4dd2-4a7d-9bbf-df9271c3fee9

Reply:
The background is here to avoid overlap between the snippet text and the copy button for long single lines (overflow-x)